backend/parser/listener/listener.go

```diff
@@ -134,6 +137,21 @@ func (s *searchListener[T]) EnterBin_op(ctx *parser.Bin_opContext) {
 }
 func (s *searchListener[T]) ExitBin_op(ctx *parser.Bin_opContext) {}
 
+func (s *searchListener[T]) EnterExists_op(ctx *parser.Exists_opContext) {}
+func (s *searchListener[T]) ExitExists_op(ctx *parser.Exists_opContext) {
+	op := strings.ToUpper(ctx.GetText())
+	switch op {
+	case "EXISTS":
+		s.currentOp = "!="
+	case "NOTEXISTS":
+		s.currentOp = "="
+	default:
+		fmt.Printf("Unknown exists operator: %s\n", op)
+	}
+
+	s.appendRules("")
```

Comment:
just checking that this will work for numeric attributes too

Reply:
just added that as a special case below - casting to a string before comparing against `''`, I think the only column this matters for on traces/logs is `Duration` atm, not sure how we want to handle `Duration = 0` but this treats it as exists